lighthouse-core/audits/time-to-interactive.js

```diff
@@ -13,9 +13,9 @@ const TracingProcessor = require('../lib/traces/tracing-processor');
 const Formatter = require('../report/formatter');
 
 // Parameters (in ms) for log-normal CDF scoring. To see the curve:
-//   https://www.desmos.com/calculator/jlrx14q4w8
-const SCORING_POINT_OF_DIMINISHING_RETURNS = 1700;
-const SCORING_MEDIAN = 5000;
+//   https://www.desmos.com/calculator/klsen8gqbf
+const SCORING_POINT_OF_DIMINISHING_RETURNS = 4000;
```

Comment:
I won't merge this, but I propose it since we're not releasing for awhile anyway and we should totally redo our scoring before we do. It's driven me nuts, if we say target = 5000, you shouldn't get a 50/100 for hitting our target. This also differentiates more between 10s (which would qualify as PWA) and say 20s.